File: PHP/Aksesoris.php
<?php
require_once 'Petshop.php'; // Mengimpor kelas Petshop

class Aksesoris extends Petshop {
    private $jenis;
    private $bahan;
    private $warna;

    // Konstruktor default
    public function __construct(
        $id = "", 
        $nama_produk = "", 
        $harga_produk = 0, 
        $stok_produk = 0, 
        $foto_produk = "",
        $jenis = "", 
        $bahan = "", 
        $warna = ""
    ) {
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk); // Konstruktor Petshop
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    // Setter
    public function set_jenis(string $jenis){ $this->jenis = $jenis; }
    public function set_bahan(string $bahan){ $this->bahan = $bahan; }
    public function set_warna(string $warna){ $this->warna = $warna; }

    // Getter
    public function get_jenis() { return $this->jenis; }
    public function get_bahan() { return $this->bahan; }
    public function get_warna() { return $this->warna; }
}
?>

File: Python/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    # Konstruktor
    def __init__(self, ID="", nama="", hargaProduk=0, stokProduk=0, jenis="", bahan="", warna=""):
        super().__init__(ID, nama, hargaProduk, stokProduk) # Konstruktor Petshop
        
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna
    
    # Setter
    def set_jenis(self, jenis):
        self.jenis = jenis
        
    def set_bahan(self, bahan):
        self.bahan = bahan
        
    def set_warna(self, warna):
        self.warna = warna
    
    # Getter
    def get_jenis(self):
        return self.jenis
        
    def get_bahan(self):
        return self.bahan
        
    def get_warna(self):
        return self.warna